src/jit/hwintrinsicxarch.cpp

```diff
@@ -179,6 +179,21 @@ static InstructionSet lookupInstructionSet(const char* className)
             return InstructionSet_POPCNT;
         }
     }
+    else if (className[0] == 'V')
+    {
+        if (strncmp(className, "Vector128", 9) == 0)
+        {
+            return InstructionSet_Vector128;
+        }
+        else if (strncmp(className, "Vector256", 9) == 0)
+        {
+            return InstructionSet_Vector256;
+        }
+        else if (strncmp(className, "Vector64", 8) == 0)
```

Comment:
I didn't think we had any `Vector64` intrinsics, even helpers, for x86 today?

Reply:
With the table-driven approach, we need to recognize this "class name" in this step. 
`Vector64` helpers would not be recognized to intrinsics because the table does not have `Vector64` rows.

Reply:
Why do we need to recognize the class name on x86 though? `InstructionSet_Vector64` isn't used anywhere on x86 and there isn't any `IsSupported` flag so it will never need to be supported.

Reply:
Because X86 side is using the tuple `("functionName", ISA_ID)` to accelerate the intrinsic recognition. So, we need to provide `("functionName", InstructionSet_Vector64)` to unify.

Reply:
I'm still not understanding.

My thinking is:
* There are no `InstructionSet_Vector64` entries for x86
* So on x86 you shouldn't need to define `InstructionSet_Vector64`
* So you shouldn't need this logic for `className == "Vector64"` in `lookupInstructionSet`

Looking at the code:
* `InstructionSet_Vector64` is only used by `lookupInstructionSet` on x86
* `lookupInstructionSet` is only used by `HWIntrinsicInfo::lookupIsa`
* `HWIntrinsicInfo::lookupIsa` (that takes the className) is only used by `HWIntrinsicInfo::lookupId`
* `HWIntrinsicInfo::lookupId` is only called from `Compiler::lookupNamedIntrinsic` in the importer

Given the above and that we will eventually need to support the ARM `IsSupported` ISAs as intrinsics returning `false`, I would expect the better fix would be to treat this as a distinct `InstructionSet_Unsupported` (or something similar).
We can then have it such that the `default` case in `lookupInstructionSet` returns this and the higher calls eventually propogate this to be `NI_ILLEGAL` so that the importer causes it to hit the software fallback.

Reply:
Changed to use `NI_ILLEGAL` instead, and removed an assertion.